google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TimelineSample.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2015 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery;
+
+import java.io.Serializable;
+import com.google.api.services.bigquery.model.QueryTimelineSample;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
+
+
+public class TimelineSample implements Serializable {
```

Comment:
Let's use AutoValue for this one.

[ViewDefinition](https://github.com/GoogleCloudPlatform/google-cloud-java/blob/7ea03518f260ceade5c464008ec44610feecd951/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ViewDefinition.java) looks to be a quite similar example in the BigQuery library.